sdks/java/io/google-cloud-platform/src/test/java/org/apache/beam/sdk/io/gcp/bigquery/BigQueryUtilsTest.java

```diff
@@ -410,96 +415,49 @@ public void testSubMilliPrecisionRejected() {
     assertThrows(
         "precision",
         IllegalArgumentException.class,
-        () ->
-            BigQueryUtils.convertAvroFormat(
-                Schema.Field.of("dummy", Schema.FieldType.DATETIME).getType(),
-                1000000001L,
-                REJECT_OPTIONS));
+        () -> BigQueryUtils.convertAvroFormat(FieldType.DATETIME, 1000000001L, REJECT_OPTIONS));
   }
 
   @Test
   public void testMilliPrecisionOk() {
     long millis = 123456789L;
     assertThat(
-        BigQueryUtils.convertAvroFormat(
-            Schema.Field.of("dummy", Schema.FieldType.DATETIME).getType(),
-            millis * 1000,
-            REJECT_OPTIONS),
+        BigQueryUtils.convertAvroFormat(FieldType.DATETIME, millis * 1000, REJECT_OPTIONS),
         equalTo(new Instant(millis)));
   }
 
   @Test
   public void testSubMilliPrecisionTruncated() {
     long millis = 123456789L;
     assertThat(
-        BigQueryUtils.convertAvroFormat(
-            Schema.Field.of("dummy", Schema.FieldType.DATETIME).getType(),
-            millis * 1000 + 123,
-            TRUNCATE_OPTIONS),
+        BigQueryUtils.convertAvroFormat(FieldType.DATETIME, millis * 1000 + 123, TRUNCATE_OPTIONS),
         equalTo(new Instant(millis)));
   }
 
   @Test
-  public void testSubMilliPrecisionLogicalTypeRejected() {
```

Comment:
Is this test not relavent anymore ?

Reply:
Yes, it's not relavent anymore. ZetaSQL TIME type is now able to support micro-second precision.